internal/pkg/statemanager/manager.go

```diff
@@ -0,0 +1,74 @@
+package statemanager
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/merico-dev/stream/internal/pkg/backend"
+)
+
+// Manager knows how to manage the States.
+type Manager interface {
+	backend.Backend
+	// GetStates is used for generate data to Write to the backend.
+	GetStates() States
+	// SetStates is used for initialize States by the data Read from the backend.
+	SetStates(states States)
+	GetState(name string) *State
```

Comment:
I added some comments at GetStates() & SetStates() but others have not is 
because the usage of these two methods is not the same as the other methods below, these two are related to backend, and the following are all memory operations, so I'm worried that users will be confused here.

Reply:
Good point. In this case, maybe the name should be more different. Let's have a short discussion together, with @lfbdev as well.

Reply:
Let's keep it for now and just add a blank line after SetStates() first.